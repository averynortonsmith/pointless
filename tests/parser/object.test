
x = {foo = "bar"}
x = {
  foo = "bar"
  n = 123;
}
x = {a = 1; b = 2}