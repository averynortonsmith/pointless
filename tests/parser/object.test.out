(Program 
 null
 []
 (Def 
  (Name "x")
  (Object 
   (Def 
    (Name "foo")
    (String "bar"))))
 (Def 
  (Name "x")
  (Object 
   (Def 
    (Name "foo")
    (String "bar"))
   (Def 
    (Name "n")
    (Number 123))))
 (Def 
  (Name "x")
  (Object 
   (Def 
    (Name "a")
    (Number 1))
   (Def 
    (Name "b")
    (Number 2)))))
